### ISSUE TITLE
Keyboard shortcuts only work after focusing certain page elements.

### ISSUE THREAD

**Issue by Arnei** [elan-ev/opencast-editor#266]
Before being able to use keyboard shortcuts like cutting or playing the video, you seemingly have to first focus the correct part of the webpage to be able to do so.

For example, the shortcut for "cutting" only works if you've focused the film strip. But if you've focused the film strip you cannot play the video by shortcut anymore. -Wien

**Comment by Arnei** [comment]
I am struggling to replicate this issue. Any further information would be appreciated.

**Comment by ziegenberg** [comment]
The problem cannot be reproduced anymore. So I propose to close the issue. It might be related to #305.

**Comment by Arnei** [comment]
If I recall correctly, you were the one that brought this issue to my attention, so if it works for you now I'm fine with closing this.
